### PR TITLE
Add kubernetes metadata for each node

### PR DESCRIPTION
**Pull request by lazam**
Closes https://github.com/paritytech/testnet-manager/issues/87

Sample of new node page:
image

image

### CODE REVIEW

File: app/lib/kubernetes_client.py [hunk 1]
```diff
@@ -5,6 +5,104 @@
...
+        'request_cpu': container_resources.requests.get('cpu', '') if container_resources.requests else 'None',
+        'request_memory': container_resources.requests.get('memory', '') if container_resources.requests else 'None',
+        'restart_count': pod.status.container_statuses[0].restart_count,
+        'tolerations': pod.spec.tolerations or [],
+        'volume_details': volume_details
+    }
+    return pod_details
+
+
+def get_pod_pvc_volume_size(namespace, pvc_name):
```
Comment (PierreBesson) [inline app/lib/kubernetes_client.py:62]: Renameto `get_pod_pvc_details`

File: app/templates/node_info.html [hunk 2]
```diff
@@ -225,14 +226,148 @@ <h3>{{ node.name }}</h3>
...
         src="/static/libs/jquery-3.6.0.slim.min.js"
         integrity="sha256-u7e5khyithlIdTpu22PHhENmPcRdFiHRjhAuHcs05RI="
         crossorigin="anonymous"></script>
 <script type="text/javascript" charset="utf8" src="/static/libs/datatables.min.js"></script>
 <script>
     $(document).ready( function () {
-        $('#table').DataTable({
+        $('#table_substrate, #table_kubernetes').DataTable({
             // https://datatables.net/reference/option/
             "lengthMenu": [[-1], ["All"]]
```
Comment (PierreBesson) [inline app/templates/node_info.html:372]: You can set a fixed size of the left column with:
```
            // https://datatables.net/reference/option/
            "lengthMenu": [[-1], ["All"]],
            "columnDefs": [
              { "width": "10%", "targets": 0 }
            ]
        });
```

**Review by BulatSaif — APPROVED** [review]

**Comment by lazam** [comment]
> Looks good! small fixes:
> 
>     * can you check why ci `smoke_test` fails?
> 
>     * can you add kernel version if it is available?
I pushed a new commit to include kernel version, and the CI should work now since I merged the helm chart changes

File: app/lib/kubernetes_client.py [hunk 3]
```diff
@@ -5,6 +5,104 @@
...
+        'phase': pvc.status.phase
+    }
+    return pvc_details
+
+
+def get_pod_node_details(pod_name):
+    node = kubernetes_client.CoreV1Api().read_node(name=pod_name)
+    node_details = {}
+
+    for label in ["beta.kubernetes.io/arch", "beta.kubernetes.io/instance-type", "beta.kubernetes.io/os"]:
```
Comment (PierreBesson) [inline app/lib/kubernetes_client.py:76]: Maybe also add the non-beta labels to this list as it those are apparently deprecated already: https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-arch

**Review by PierreBesson — CHANGES_REQUESTED** [review]

File: app/templates/node_info.html (hunk 2 above)
Comment (lazam) [reply app/templates/node_info.html:372]: This is what I'm looking for! Thanks!

File: app/lib/kubernetes_client.py (hunk 3 above)
Comment (lazam) [reply app/lib/kubernetes_client.py:76]: Nice! I removed the deprecated values and replaced with the new one

**Pull request merged**